### DISCUSSION TITLE
DB 추가 작업 정리

### DISCUSSION THREAD

**Discussion by liebertar** [bitcamp-homies/caffeine-frontend#31] [Ideas]
Team leader가 매우 바쁜 관계로 
이번에 DB에 추가적으로 들어갈 지역 / 카페 정보 대신 정리 드립니다.

1. 배정 지역 (25개씩)

박연수 : 양천구, 도봉구
김정구 : 동대문구, 중랑구
김수정 : 구로구, 송파구
양웅비 : 노원구, 서대문구
김대일 : 동작구, 금천구
이창근 : 종로구,  은평구
이풍혁 : 영등포구, 강서구

2. 주의 사항

- 각 row 에 하기 내용 포함해주시기 바랍니다.
    (1) 인스타 계정 
    (2) 카페명(되도록 국문 사용) 
    (3) 주소 1(서울) 
    (4) 주소 2(성동구) 
    (5) 주소 3(서울숲6길 19) 
    (6) 주소 4(층수 : null 가능) 
    (7) 소개글 (insta profile)
    (8) 사진 (profile 1장, 기타 4장)

- null 값은 주소 4만 허용입니다.

하기 주소에 구별로 이름(해당지역) tab 생성하시어 작업해주시면 됩니다!
https://docs.google.com/spreadsheets/d/1Dd9000BpfHSo5EJlXVJ7zRzWxqjIgGHrvT_DjVZkI_g/edit?usp=sharing

건의 하실 내용이나 Idea share 편하게 부탁드립니다.

**Comment by sudong1004** [comment]
확인했습니다~
